python-checks/src/test/resources/checks/numpyIsNan2.py

```diff
@@ -0,0 +1,14 @@
+import numpy as xx
+
+
+def foo(x):
+    if x == xx.nan: print(1)  # Noncompliant
```

Comment:
Sorry, I missed it the first time. We are missing the check for the noncompliant message 
````
if x == xx.nan: print(1) # Noncompliant {{Equality checks should not be made against \"numpy.nan\". Use numpy.isnan() instead.}}
```

Reply:
No need to be sorry. You didn't miss anything, b80fd05f99ef4c801e5dbdb6b3be08d39b8966b5 was a response to your comment :)

Reply:
Yes agree, I do not think we should check for this. I think it is more than enough what you already did, it covers all the main cases.

Reply:
Addressed in 51ce44553971f9b57e200745beb6fcfea817d406